### PR TITLE
refactor: fix typo operators

### PR DESCRIPTION
**Pull request by awjae**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jung-han** [comment]
ok to test

File: package-lock.json [hunk 1]
```diff
@@ -3352,8 +3352,7 @@
       "version": "1.3.0",
```
Comment (jung-han) [inline package-lock.json:3352]: Is it content that needs to change dependence?

Comment (jung-han) [reply package-lock.json:3352]: If it is not related to a typo, it would be nice to remove it and upload it!

File: package-lock.json [hunk 2]
```diff
@@ -1,10436 +0,0 @@
-{
```
Comment (jung-han) [inline package-lock.json:1]: Rather than deleting the file, it seems necessary to make it the initial state so that there are no changes.

Comment (awjae) [reply package-lock.json]: Well.. I am not familiar with how to use git...  
is this right?

**Review by jung-han — APPROVED** [review]
Thanks for contribution 🌝

**Pull request merged**